orm/query.go

```diff
@@ -219,29 +220,44 @@ func (q *Query) AllWithDeleted() *Query {
 
 // With adds subq as common table expression with the given name.
 func (q *Query) With(name string, subq *Query) *Query {
-	return q._with(name, newSelectQuery(subq))
+	return q._with(name, newSelectQuery(subq), false)
+}
+
+func (q *Query) WithRecursive(name string, subq *Query) *Query {
```

Comment:
Could you add just a very brief comment here like you have below in the wrap function.